src/open_inwoner/kvk/tests/mocks.py

```diff
@@ -24,61 +24,103 @@
     ],
 }
 
-multiple = {
+hoofdvestiging = {
     "pagina": 1,
-    "aantal": 10,
+    "resultatenPerPagina": 10,
+    "totaal": 1,
+    "resultaten": [
+        {
+            "kvkNummer": "68750110",
+            "vestigingsnummer": "000037178598",
+            "naam": "Test BV Donald",
+            "adres": {
+                "binnenlandsAdres": {
+                    "type": "bezoekadres",
+                    "straatnaam": "Hizzaarderlaan",
+                    "plaats": "Lollum",
+                }
+            },
+            "type": "hoofdvestiging",
+            "_links": {
+                "basisprofiel": {
+                    "href": "https://api.kvk.nl/test/api/v1/basisprofielen/68750110"
+                },
+                "vestigingsprofiel": {
+                    "href": "https://api.kvk.nl/test/api/v1/vestigingsprofielen/000037178598"
+                },
+            },
+        }
+    ],
+    "_links": {
+        "self": {
+            "href": "https://api.kvk.nl/test/api/v2/zoeken?kvknummer=68750110&pagina=1&resultatenperpagina=10&type=hoofdvestiging"
+        }
+    },
+}
+
+multiple_branches = {
+    "pagina": 1,
+    "resultatenPerPagina": 10,
     "totaal": 3,
     "resultaten": [
         {
             "kvkNummer": "68750110",
-            "handelsnaam": "Test BV Donald",
+            "naam": "Test BV Donald",
```

Comment:
I do see that `mocks.simple` still uses `handelsnaam` https://github.com/maykinmedia/open-inwoner/blob/upgrade/2069-kvk/src/open_inwoner/kvk/tests/mocks.py#L15 (same thing for `aantal` instead of `resultatenPerPagina`)

Let's also use the v2 schema here to be consistent